src/tests/asyncTests.ts

```diff
@@ -5,7 +5,7 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import { ConfigAggregator, Connection } from '@salesforce/core';
+import { Connection } from '@salesforce/core';
```

Comment:
newbie question: what is the ConfigAggregator? what is it different from using it vs not using it?